payjoin/tests/integration.rs

```diff
@@ -879,6 +932,13 @@ mod integration {
         Ok(payjoin_psbt.extract_tx()?)
     }
 
+    fn predicted_tx_weight(tx: &bitcoin::Transaction) -> Weight {
+        bitcoin::transaction::predict_weight(
+            vec![bitcoin::transaction::InputWeightPrediction::P2WPKH_MAX; tx.input.len()],
```

Comment:
I find it mildly irritating that we have to assume P2WPKH script since the `tx` primitive doesn't have prevout data, but I'm OK living with it. PSBTv2 fixes this.